### ISSUE TITLE
Dropdown in dropdown

### ISSUE THREAD

**Issue by kkaplita** [Dogfalo/materialize#5660]
When use dropdown inside other dropdown... first close automatic before open second dropdown. In old materialize was option preventDefault...

## Expected Behavior
When open second dropdown (in other dropdown)... first shouldn't close.

## Current Behavior
First dropdown closing...

## Possible Solution
Add preventDefault option

**Comment by acburst** [comment]
Have you tried the closeOnClick option?

**Comment by kkaplita** [comment]
Of course, I tried this - not working. If you click some else element in first dropdown - everything is ok but when you click on other dropdown (second dropdown in first one)... First instance close automatically.
